code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -683,19 +680,48 @@
 /datum/reagent/consumable/maltodextrin
 	name = "Maltodextrin"
 	description = "A common filler found in processed foods. Foods containing it will leave you feeling full for a much shorter time."
-	color = "#ffffff"
+	color = "#d4e1ee"
 	chem_flags = CHEMICAL_RNG_GENERAL
 	taste_mult = 0.1 // Taste the salt and sugar not the cheap carbs
 	taste_description = "processed goodness"
 	nutriment_factor = 0
 	metabolization_rate = 0.05 * REAGENTS_METABOLISM //Each unit will last 50 ticks
+	overdose_threshold = 5
```

Comment:
I really don't think this should use the overdose system because of the big red warning that shows up in chat naming the exact reagent. This is almost more of an issue with the overdose system than anything, but regardless I don't like how this will play out.


# *YOU FEEL LIKE YOU HAD TOO MUCH OF MALTODEXTRIN*

At the very least the overdose threshold should be higher so that players are only like to see it after repeated junk food binging instead of during their first eating session. Either double this to `10` or remove it entirely and make it so that the upset stomach happens after `current_cycle` reaches a certain amount.